cityofaustin/atd-moped
# [BUG] Unable to edit project project funding without populating most fields

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/15314

## Testing
**URL to test:** 
https://deploy-preview-1249--atd-moped-main.netlify.app/moped/ 

**Steps to test:**
1. Go to the project Funding tab.
2. Add a new funding source, fill in all the fields.
3. Now edit the funding source and delete/clear out all the columns that allow you to clear or delete. You should be able to save without error.
4. If you type in a whole bunch of spaces to the description field it should just save as null instead of saving empty string


---
#### Ship list
